pages/static/containers_js.go

```diff
@@ -468,6 +468,30 @@ function drawFileSystemUsage(machineInfo, stats) {
 	}
 }
 
+// Draw the graph for Custom Metrics.
```

Comment:
Assumption: PollingFrequency is same for all custom metrics which leads to custom_metrics.length being same in all stats corresponding to different timestamps.